Gruntfile.js

```diff
@@ -63,55 +36,42 @@ module.exports = function(grunt) {
     },
 
     rollup: {
-      cjs: {
```

Comment:
I'd like to highlight that the CommonJS distribution file has been removed and is no longer generated by the build since Node.js modules will now import the `src/runtime/node.js` module directly as it is the new "main" entry in `package.json`.